llms/mlx_lm/models/qwen2.py

```diff
@@ -51,6 +52,8 @@ def __init__(self, args: ModelArgs):
         self.v_proj = nn.Linear(dim, n_kv_heads * head_dim, bias=True)
         self.o_proj = nn.Linear(n_heads * head_dim, dim, bias=False)
 
+        assert args.rope_scaling is not None
```

Comment:
The test is failing because its ok for this to be `None`. Do you mind to fix it?

Reply:
Sorry. Will do

Reply:
Done! I will make sure that I run unit test locally before filing a pull request in the future.